src/Shared/NcdfUtil/hco_ncdf_mod.F90

```diff
@@ -413,6 +413,18 @@ SUBROUTINE NC_READ_TIME( fID,     nTime,        timeUnit, &
     ! Read calendar attribute
     IF ( PRESENT( timeCalendar ) ) THEN
        CALL NcGet_Var_Attributes( fId, v_name, 'calendar', timeCalendar )
```

Comment:
What happens if the calendar attribute is not in the file?

Reply:
Currently nothing.  I didn't know if we wanted to always force an error if we have no calendar.  Most people usually assume "standard".